### ISSUE TITLE
Export not properly working

### ISSUE THREAD

**Issue by chri4354** [empiricaly/empirica#60]
I run my significant experiment yesterday (N=150) and tried to export the data going to meteorapp-name.com/admin/export.csv that worked with a previously run pilot. However, perhaps due to the large volume of participants, this time the export did not work. On Chrome, the download starts and after a few minutes it fails with the message "Failed - Network error". On Firefox, the download completes, but when I open the file, it contains way too few lines (10 times less than expected). The last line is usually half written (ie not all columns have been filled) as if the download suddenly stopped.

**Comment by npaton** [comment]
I've pushed hopefully a fix for this. It's hard to tell exactly if it is a fix for your case, but it should help. Please let me know if you are still seeing this issue on export.

**Comment by amaatouq** [comment]
@chri4354 did this fix your problem?

PS: @npaton, now I have created a private repo for the `global_vs_local` experiment on my account (rather than the MIT annoying one) and added you as a collaborator: https://github.com/amaatouq/global_vs_local

**Comment by chri4354** [comment]
Do I need to redeploy the app to know if this works?

**Comment by amaatouq** [comment]
yes--redeploying from the new repo (i.e., moving your settings.json file to the folder you cloned from this url https://github.com/amaatouq/global_vs_local ) .. if this solves your issue, then we can close it.

**Comment by chri4354** [comment]
@npaton did you push the fix to my repo or @amaatouq 's? I want to make sure the two versions are on the same page before creating conflicting versions.

**Comment by amaatouq** [comment]
the fix was on empirica (the master branch). I merged the latest version (that includes the fix) into my repo of the experiment.

**Comment by npaton** [comment]
Ok. At least it's no longer making Atlas send an alert. But it's still seems to be asking too much of that cursor. I'm going to need to debug with the data. @amaatouq Do you allow me to use a dump of the database on Atlas to debug this issue?

**Comment by amaatouq** [comment]
Of course, go ahead please.
